src/defaultPropTypes.js

```diff
@@ -17,11 +17,11 @@ export default {
   activeBarStyle: PropTypes.object,
   dismissAfter: PropTypes.oneOfType([
     PropTypes.bool,
-    PropTypes.number
+    PropTypes.number,
```

Comment:
@pburtchaell seconded - we should sweep through our lint configuration or decide on a standard?